client/app/components/app-header/index.js

```diff
@@ -1,58 +0,0 @@
-import debug from 'debug';
-import CreateDashboardDialog from '@/components/dashboards/CreateDashboardDialog';
-
-import logoUrl from '@/assets/images/redash_icon_small.png';
-import frontendVersion from '@/version.json';
-import template from './app-header.html';
-import './app-header.css';
-
-const logger = debug('redash:appHeader');
-
-function controller($rootScope, $location, $route, $uibModal, Auth, currentUser, clientConfig, Dashboard, Query) {
-  this.logoUrl = logoUrl;
-  this.basePath = clientConfig.basePath;
-  this.currentUser = currentUser;
-  this.showQueriesMenu = currentUser.hasPermission('view_query');
-  this.showAlertsLink = currentUser.hasPermission('list_alerts');
```

Comment:
I noticed you didn't port this. Please make sure that you port the other permissions correctly.